.npmrc

```diff
@@ -0,0 +1,14 @@
+# Enforce engine constraints
+engine-strict=true
+
+# Use frozen lockfile in CI and production
+frozen-lockfile=true
+
+# Use the specified Node.js version
+use-node-version=22.21.0
+
+# Don't install packages published within the last week (7 days = 604800000 ms)
+publish-age=604800000
+
+# Enable postinstall scripts for packages that need them
+enable-pre-post-scripts=true
```

Comment:
This was not the setting I was intending. Please remove it, run install, and if any packages wanted to run post install scripts, we can add them explicitly to a list of packages allowed to run